### PR TITLE
Add zola

### PR DESCRIPTION
**Pull request by salasrod**

### CODE REVIEW

**Comment by salasrod** [comment]
![Build Status](https://github.com/salasrod/asdf-zola/workflows/CI/badge.svg

File: plugins/zola [hunk 1]
```diff
@@ -0,0 +1 @@
+repository = https://github.com/salasrod/asdf-zola
```
Comment (imbsky) [inline plugins/zola]: This should be the git url

```suggestion
repository = https://github.com/salasrod/asdf-zola.git
```

**Comment by imbsky** [comment]
Looks good overall.

**Comment by imbsky** [comment]
Thanks @salasrod for your contribution!

**Comment by salasrod** [comment]
> 
> 
> Thanks @salasrod for your contribution!

Thank you for reviewing it!

**Comment by imbsky** [comment]
Anytime!

**Pull request merged**